.github/workflows/ci-haddock.sh

```diff
@@ -1,7 +1,7 @@
 #!/usr/bin/env bash
 set -e
 
-cabal haddock --haddock-tests all
+cabal haddock --disable-documentation --haddock-tests all
```

Comment:
Should: leave a comment why the heck we would want to `--disable-documentation` here.

I was out of the loop on that finding and leaving a hint/link here seems like a good idea to me.

Reply:
It's in the commit message but you're right, let's put it in the code :)